js/service/FileService.js

```diff
@@ -80,8 +80,8 @@ export default class FileService {
 			});
 			if (typeof existingFile !== 'undefined') {
 				OC.dialogs.confirm(
-					`A file with the name ${fileItem.file.name} already exists. Do you want to overwrite it?`,
-					'File already exists',
+					t('deck', 'A file with the name ${fileItem.file.name} already exists. Do you want to overwrite it?'),
```

Comment:
```suggestion
					t('deck', `A file with the name ${fileItem.file.name} already exists. Do you want to overwrite it?`),
```

Otherwise the variable is not resolved